cassiopeia/simulator/TreeSimulator.py

```diff
@@ -5,15 +5,32 @@
 implement a method called `simulate_tree`.
 """
 import abc
+
 from cassiopeia.data import CassiopeiaTree
 
 
 class TreeSimulator(abc.ABC):
     """
     TreeSimulator is an abstract class that all tree simulators derive from.
+
+    A TreeSimulator returns a CassiopeiaTree whose tree topology is initialized.
```

Comment:
"whose tree topology is initialized" -- are we saying that a TreeSimulator should _not_ have branch lengths on the tree?

I see below we say that some `TreeSimulators` can return a tree with branch lengths -- why don't we make this slightly more clear: "A TreeSimulator returns a CassiopeiaTree with _at least_ its tree topology initialized"

Reply:
Great suggestion!